book/07-git-tools/sections/stashing-cleaning.asc

```diff
@@ -281,6 +281,6 @@ This way you can step through each file individually or specify patterns for del
 [NOTE]
 ====
 There is a quirky situation where you might need to be extra forceful in asking Git to clean your working directory. 
-It's possible that you have a working directory wherein one or more subdirectories are _themselves_ Git clones, which means you have subdirectories that contain their own `.git` directories.
-Even with the "force" `-f` option, Git will not remove ignored subdirectories that contain either a file or directory named `.git` -- to get rid of those, you need to use _two_ `-f` options for emphasis.
+If you happen to be in a working directory under which you've copied or cloned other Git repositories (perhaps as submodules), even `git clean -fd` will refuse to delete those directories.
```

Comment:
Hmm, I hate bringing submodules into it like this, but I guess that's probably the primary use case for having two `-f` flags.